drivers/clock_control/nrf_power_clock.c

```diff
@@ -228,6 +228,21 @@ static struct clock_control_async_data *list_get(sys_slist_t *list)
 	return async_data;
 }
 
+static inline void anomaly_132_workaround(nrf_clock_task_t start_tsk)
+{
+	static bool once;
+
+	#ifndef CONFIG_NRF52_ANOMALY_132_DELAY_US
+	#define CONFIG_NRF52_ANOMALY_132_DELAY_US 0
+	#endif
+
+	if ((start_tsk == NRF_CLOCK_TASK_LFCLKSTART) && !once &&
```

Comment:
Just curious. Is the generated code shorter when the task is checked? It feels more natural to check `type` (`subsys`) instead.

Reply:
change to subsys, it's better and cleaner.